.github/workflows/docker-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: Continuous Build Docker
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build-zipkin-gcp:
+    name: Build Zipkin GCP
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 10
+      # Remove apt repos that are known to break from time to time.
+      # See https://github.com/actions/virtual-environments/issues/323
+      - name: Remove broken apt repos [Ubuntu]
+        run: |
+          for apt_file in `grep -lr microsoft /etc/apt/sources.list.d/`; do sudo rm $apt_file; done
+      # Setup latest JDK. We do this to ensure users don't need to use the same version as our
+      # release process. Release uses JDK 11, the last version that can target 1.6 bytecode.
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15
+      - name: Cache NPM Packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: npm-packages-${{ runner.os }}-${{ hashFiles('zipkin-lens/package-lock.json') }}
+      - name: Cache Maven Modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: m2-repository-${{ hashFiles('**/pom.xml') }}
+      - name: Cache Maven Wrapper
```

Comment:
Not related to this PR just observation since I only realized it now, I wonder if it's idiomatic to check the wrapper into the repo?

https://github.com/takari/maven-wrapper/tree/master/.mvn/wrapper

Reply:
we did then we reverted in in our apache experiment as there was a fuss about binaries in git. we could have renamed it to jpg to get around that :D

meanwhile I suppose we could do that as it is simpler than all these little stages, and rarely changes.

Reply:
nuked it here and in zipkin. the topic is indeed distracting.. there are source files larger than that binary, I'd bet :P